cmd/sync/flag.go

```diff
@@ -13,21 +13,23 @@ import (
 )
 
 const (
-	flagDstRegistryName     = "dst-name"
-	flagDstRegistryUser     = "dst-user"
-	flagDstRegistryPassword = "dst-password"
-	flagLastModified        = "last-modified"
-	flagLoop                = "loop"
-	flagMetricsPort         = "metrics-port"
+	flagDstRegistryName         = "dst-name"
+	flagDstRegistryUser         = "dst-user"
+	flagDstRegistryPassword     = "dst-password"
+	flagLastModified            = "last-modified"
+	flagLoop                    = "loop"
+	flagSyncPrivateRepositories = "sync-private-repositories"
```

Comment:
Also, I thought about making this registry specific, but with e.g. Azure container registry - it can also support private images. It is just not possible with anonymous access.